packages/app-project/stores/User/UserPersonalization/Notifications/Notifications.js

```diff
@@ -0,0 +1,85 @@
+import { flow, getRoot, types } from 'mobx-state-tree'
+import { sugarClient } from 'panoptes-client/lib/sugar'
+import talkClient from 'panoptes-client/lib/talk-client'
```

Comment:
Would it be a lot of work to use the new API client to get resources from the Talk API?